packages-web/badge/src/components/__tests__/Badge.spec.ts

```diff
@@ -8,9 +8,9 @@ describe("Badge", () => {
 
     it("should render the structure", () => {
         const badgeProps: BadgeProps = {
-            badgeType: "badge",
```

Comment:
name is missing

Reply:
What name?

Reply:
property name..

Reply:
There is no name property. Do you mean className?

Reply:
Confusing with native 😰